testing/grass/APKBUILD

```diff
@@ -0,0 +1,152 @@
+# Contributor: Pietro Zambelli <peter.zamb@gmail.com>
+# Maintainer: Pietro Zambelli <peter.zamb@gmail.com>
+pkgname=grass
+pkgver=7.8.0
+pkgrel=0
+pkgdesc="GRASS GIS is a suite used for geospatial analysis, image processing and spatial modeling."
+url="https://grass.osgeo.org/"
+arch="all"
+license="GPL-2.0-or-later"
+depends="
+	attr
+	bash
+	bison
+	bzip2
+	cairo
+	fftw
+	flex
+	freetype
+	gdal
+	gettext
+	geos
+	gnutls
+	libbz2
+	libjpeg-turbo
+	libpng
+	musl
+	musl-utils
+	ncurses
+	openjpeg
+	openblas
+	py3-numpy
+	py3-pillow
+	py3-six
+	postgresql
+	sqlite
+	sqlite-libs
+	tiff
+	wxgtk-base
+	wxgtk
+	zstd
+	zstd-libs
+	"
+makedepends="
+	build-base
+	bzip2-dev
+	cairo-dev
+	fftw-dev
+	freetype-dev
+	gcc
+	gdal-dev
+	geos-dev
+	gnutls-dev
+	libc6-compat
+	libjpeg-turbo-dev
+	libpng-dev
+	make
+	openjpeg-dev
+	openblas-dev
+	postgresql-dev
+	python3-dev
+	py3-numpy-dev
+	sqlite-dev
+	tar
+	tiff-dev
+	unzip
+	wxgtk-base-dev
+	wxgtk-dev
+	zip
+	zstd-dev
+	"
+subpackages="
+	$pkgname-dev:dev
+	$pkgname-doc:doc
+	$pkgname-gui:gui
+	$pkgname-fonts:fonts
+	$pkgname-demolocation:demo
+	"
+
+source="${pkgname}-${pkgver}.tar.gz::https://grass.osgeo.org/$pkgname`echo ${pkgver::-2} | tr -d .`/source/$pkgname-$pkgver.tar.gz"
```

Comment:
You would probably be better adding an additional variable like this to remove the periods
`_pkgver=${pkgver//.}`
You can then just reference your source line like this
`source="$pkgname-$pkgver.tar.gz::https://grass.osgeo.org/grass${_pkgver::-1}/source/grass-$pkgver.tar.gz"`

Note that I have remove the $pkgname in the source url too